### PR TITLE
EZP-29254: Images embedded in RichText editor dissapear after undo

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-29254

The timeout is for multiple `cmd-z` clicks. I think 1sec is ok, but I'm open to suggestions.

### CODE REVIEW

File: Resources/public/js/alloyeditor/plugins/embed.js [hunk 1]
```diff
@@ -86,6 +88,31 @@ YUI.add('ez-alloyeditor-plugin-embed', function (Y) {
...
+                 * @private
+                 */
+                _attachUndoListener: function () {
+                    if (isUndoEventAttached) {
+                        return;
+                    }
+
+                    isUndoEventAttached = true;
+
+                    editor.on('afterCommandExec', function(e) {
```
Comment (sunpietro) [inline Resources/public/js/alloyeditor/plugins/embed.js]: `event` please

File: Resources/public/js/alloyeditor/plugins/embed.js [hunk 2]
```diff
@@ -86,6 +88,31 @@ YUI.add('ez-alloyeditor-plugin-embed', function (Y) {
...
+                        if (e.data.name === 'undo') {
+                            window.clearTimeout(reloadEmbedTimeout);
+
+                            reloadEmbedTimeout = window.setTimeout(function() {
+                                editor.fire('snapshotRestored');
+                            }, 1000);
```
Comment (sunpietro) [inline Resources/public/js/alloyeditor/plugins/embed.js]: Why is it 1 second?

File: Resources/public/js/alloyeditor/plugins/embed.js (hunk 1 above)
Comment (dew326) [reply Resources/public/js/alloyeditor/plugins/embed.js]: I want to stay in line with this file and Platform patterns ;)

**Comment by barbaragr** [comment]
1. Go to Content/Content structure
2. Click Create and choose Image
3. Fill required fields and click Publish

Actual result: 
There is an error: `An error occurred while publishing the draft` Console shows:
`ErrorMessage: {_media-type: "application/vnd.ez.api.ErrorMessage+json", errorCode: 500,…}}
ErrorMessage:
{_media-type: "application/vnd.ez.api.ErrorMessage+json", errorCode: 500,…}
errorCode: 500 errorDescription:
"Attempted to call an undefined method named "parseContentRequest" of class "EzSystems\PlatformUIBundle\Controller\Rest\ContentController"."
errorMessage:"Internal Server Error"
file
:"/var/www/ezv1/vendor/ezsystems/platform-ui-bundle/Controller/Rest/ContentController.php"
line:33
trace:"#0 /var/www/ezv1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(135): EzSystems\PlatformUIBundle\Controller\Rest\ContentController->createContent(Object(Symfony\Component\HttpFoundation\Request))↵#1 /var/www/ezv1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(57): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)↵#2 /var/www/ezv1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php(67): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)↵#3 /var/www/ezv1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(183): Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)↵#4 /var/www/ezv1/web/app.php(73): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))↵#5 {main}"
_media-type:"application/vnd.ez.api.ErrorMessage+json"`

I've checked in on v1.7.7 and it works fine.

**Comment by lserwatka** [comment]
@dew326 could you merge it up?

**Comment by dew326** [comment]
@lserwatka done

**Comment by lserwatka** [comment]
It was a CR, so I guess we need a patches.

**Comment by dew326** [comment]
I'm on it.

**Pull request merged**